spec/shopify_transporter/shopify_transporter_spec.rb

```diff
@@ -306,6 +329,94 @@ def tmpfile(content)
     end
   end
 
+  describe ShopifyTransporter::Exporters::Exporter do
```

Comment:
Any reason why this is not in its own test file?

Reply:
Only because all the tests for other analogous commands are in this file and I didn't want to break away from the established pattern. Happy to move them out.